packages/health-check/src/utils.js

```diff
@@ -61,6 +61,19 @@ function getRequiredEnvironmentVariable(name) {
   return value;
 }
 
+/**
+ * Fetches test user's API key from the environment. Raises an error if that
+ * is not possible.
+ * @returns {string} API key
+ */
+function getAPIKey() {
+  const apiKey = getRequiredEnvironmentVariable("ACCOUNTS_TEST_USER_API_KEY");
+  if (!apiKey) {
+    throw new Error("Missing or empty environment variable ACCOUNTS_TEST_USER_API_KEY.");
+  }
+  return apiKey;
+}
+
```

Comment:
I don't think we need this method if we've decided to set test api key in env variables, other services should use the env variable directory instead.